src/ui/server/helpers/create-date-from-numbers/index.test.ts

```diff
@@ -0,0 +1,42 @@
+import createDateFromNumbers from '.';
+
+describe('server/helpers/create-date-from-numbers', () => {
+  const day = 1;
+  const month = 12;
+  const year = 2022;
+
+  it('shouild return a formatted currency', () => {
+    const result = createDateFromNumbers(day, month, year);
+
+    const expected = new Date(`${month} ${day} ${year}`);
+
+    expect(result).toEqual(expected);
+  });
+
+  describe('when day is not provided', () => {
+    it('should return null', () => {
+      // @ts-ignore
+      const result = createDateFromNumbers(undefined, month, year);
```

Comment:
Any specific reason for instructing TS complier to ignore following lines in this file?

Reply:
The function expects day, month and year and we're testing that it returns null when one of these params is not provided. Typescript complains about field not provided so add this to ignore the TS error